lib/client.ml

```diff
@@ -1,10 +1,24 @@
 open Lwt_unix
 
+module Message = struct
+  type label =
```

Comment:
I think this type will be enrich with the ones I had defined on my PR, right?

Reply:
Yes!